src/redux/actions/authActions.js

```diff
@@ -1 +1,42 @@
-import * as types from '../types';
+import { LOGIN_SUCCESS, LOGIN_FAIL, LOGOUT, SET_LOADING } from '../types';
+import axios from 'axios';
+import store from '../store/index';
+import ActionButton from 'antd/lib/modal/ActionButton';
+
+console.log(store.getState());
+const reduxState = store.getState();
+//Login User
+export const loginUser = accessToken => async dispatch => {
+  try {
+    setLoading();
+    const res = await axios.post(
+      `${reduxState.gen.backendEndpoint}/google/auth/token/`,
+      {
+        token: accessToken,
+      },
+    );
+    dispatch({
+      type: LOGIN_SUCCESS,
+      payload: res.data,
+    });
+  } catch (err) {
+    dispatch({
+      type: LOGIN_FAIL,
+      payload: err.response.data,
+    });
+  }
+};
+
+//Logout User
+export const logoutUser = () => dispatch => {
+  dispatch({
+    type: LOGOUT,
+  });
+};
```

Comment:
Why using `dispatch` in this when `return` can be used directly?

Reply:
Since, we are altering the ```redux state``` after a user logs out. Therefore, used the dispatch.

Reply:
I guess you can use `return` directly. `dispatch` isn't making sense to me.